cmd/kured/main.go

```diff
@@ -261,9 +264,9 @@ func commandReboot(nodeID string) {
 	}
 
 	// Relies on hostPID:true and privileged:true to enter host mount space
-	rebootCmd := newCommand("/usr/bin/nsenter", "-m/proc/1/ns/mnt", "/bin/systemctl", "reboot")
+	rebootCmd := newCommand("/usr/bin/nsenter", "-m/proc/1/ns/mnt", "/bin/systemctl", command)
```

Comment:
in this case, shouldn't also systemctl be part of the "command" variable? 
In that case the default command could be "/bin/systemctl reboot". Allows more flexibility for non systemd linux systems.

Reply:
Good point!